Test failed to link on Linux

It works fine on Mac however on Ubuntu I got this error.

```Error output:

/usr/bin/ld: ../src/copentimelineio/libcopentimelineio.a(track.cpp.o):(.data+0x8): multiple definition of `TrackKind_Audio'; CMakeFiles/COTIOCompositionTests.dir/OTIOCompositionTests.c.o:(.bss+0x60): first defined here
/usr/bin/ld: ../src/copentimelineio/libcopentimelineio.a(track.cpp.o):(.data+0x0): multiple definition of `TrackKind_Video'; CMakeFiles/COTIOCompositionTests.dir/OTIOCompositionTests.c.o:(.bss+0x58): first defined here
clang: error: linker command failed with exit code 1 (use -v to see invocation)
gmake[2]: *** [tests/CMakeFiles/COTIOCompositionTests.dir/build.make:103: tests/COTIOCompositionTests] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:1341: tests/CMakeFiles/COTIOCompositionTests.dir/all] Error 2
gmake: *** [Makefile:166: all] Error 2
```

--- a/include/copentimelineio/marker.h
+++ b/include/copentimelineio/marker.h
@@ -10,21 +10,27 @@
 
 #ifdef __cplusplus
 # define OTIO_API extern "C"
+# define OTIO_API_BEGIN OTIO_API {
+# define OTIO_API_END }
 #else
 # define OTIO_API
+# define OTIO_API_BEGIN
+# define OTIO_API_END
 #endif
 
-OTIO_API const char *MarkerColor_pink;
-OTIO_API const char *MarkerColor_red;
-OTIO_API const char *MarkerColor_orange;
-OTIO_API const char *MarkerColor_yellow;
-OTIO_API const char *MarkerColor_green;
-OTIO_API const char *MarkerColor_cyan;
-OTIO_API const char *MarkerColor_blue;
-OTIO_API const char *MarkerColor_purple;
-OTIO_API const char *MarkerColor_magenta;
-OTIO_API const char *MarkerColor_black;
-OTIO_API const char *MarkerColor_white;
+OTIO_API_BEGIN
+extern const char *MarkerColor_pink;
+extern const char *MarkerColor_red;
+extern const char *MarkerColor_orange;
+extern const char *MarkerColor_yellow;
+extern const char *MarkerColor_green;
+extern const char *MarkerColor_cyan;
+extern const char *MarkerColor_blue;
+extern const char *MarkerColor_purple;
+extern const char *MarkerColor_magenta;
+extern const char *MarkerColor_black;
+extern const char *MarkerColor_white;
+OTIO_API_END
 typedef struct RetainerMarker RetainerMarker;
 typedef struct Marker Marker;
 

--- a/include/copentimelineio/track.h
+++ b/include/copentimelineio/track.h
@@ -19,8 +19,12 @@
 
 #ifdef __cplusplus
 # define OTIO_API extern "C"
+# define OTIO_API_BEGIN OTIO_API {
+# define OTIO_API_END }
 #else
 # define OTIO_API
+# define OTIO_API_BEGIN
+# define OTIO_API_END
 #endif
 
 typedef struct Track Track;
@@ -31,8 +35,10 @@ typedef enum {
 } OTIO_Track_NeighbourGapPolicy_;
 typedef int OTIO_Track_NeighbourGapPolicy;
 
-OTIO_API const char *TrackKind_Video;
-OTIO_API const char *TrackKind_Audio;
+OTIO_API_BEGIN
+extern const char *TrackKind_Video;
+extern const char *TrackKind_Audio;
+OTIO_API_END
 
 OTIO_API Track *Track_create(
         const char *name,

